.github/workflows/ci.yml

```diff
@@ -53,13 +53,11 @@ jobs:
 
   PHPUnit-hhvm:
     name: PHPUnit (HHVM)
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
+    container: hhvm/hhvm:3.30-lts-latest
     continue-on-error: true
     steps:
-      - uses: actions/checkout@v2
-      - uses: azjezz/setup-hhvm@v1
-        with:
-          version: lts-3.30
-      - run: composer self-update --2.2 # downgrade Composer for HHVM
-      - run: hhvm $(which composer) install
+      - uses: actions/checkout@v1
```

Comment:
Might have missed it, but why are we bumping this down to `v1` instead of up to `v3`? `v1` hasn't been updated in years and AFAIK contains a security issue. Where `v3` is the latest maintained version that should do what we need from it.

Reply:
Agree that we want to use `v3` for new jobs!

This change here only affects legacy HHVM (which is no longer supported and will be removed in the future). In particular, the legacy HHVM container runs on legacy Ubuntu 18.04 which ships a legacy git version. Both `v2` and `v3` use the GitHub API instead of cloning from git when detecting a legacy git version < `2.18`, but this would fail because the legacy HHVM container does not even have curl installed. As such, I've pinned this at `v1` to clone from git as expected.

Legacy is fun! I'm glad we can get rid of this mess with the current plans for [ReactPHP v3](https://github.com/orgs/reactphp/discussions/472).

Reply:
Awesome :+1:!